.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "AWS CDK & Python Development Environment",
+  "image": "mcr.microsoft.com/devcontainers/base:ubuntu-22.04",
+  "features": {
+    "ghcr.io/devcontainers/features/node:1.5.0": {
+      "version": "22.6.0"
+    },
+    "ghcr.io/devcontainers/features/python:1.6.3": {
+      "version": "3.12.0"
+    },
+    "ghcr.io/devcontainers/features/aws-cli:1": {}
+  },
+  "postCreateCommand": "npm install -g aws-cdk@2.151.0 && pip install --user -U pre-commit && pre-commit install-hooks",
```

Comment:
why use `&&`?  doesn't postCreateCommand take a list of strings?  https://github.com/devcontainers/spec/blob/main/docs/specs/devcontainer-reference.md#merge-logic

Reply:
It takes a string or an array of strings. The code is now using an array purely for esthetic reasons.